holoviews/plotting/plot.py

```diff
@@ -584,6 +597,12 @@ class GenericElementPlot(DimensionedPlot):
     bgcolor = param.ClassSelector(class_=(str, tuple), default=None, doc="""
         If set bgcolor overrides the background color of the axis.""")
 
+    default_span = param.ClassSelector(default=2.0, class_=(int, float, tuple), doc="""
+        Defines the span of an axis if the axis range is zero, i.e. if
+        the lower and upper end of an axis are equal. For example if
+        there is a single datapoint at 0 a default_span of 2.0 will
+        result in axis ranges spanning from -1 to 1.""")
```

Comment:
The equivalent of `default_span` in datashader is used both when min==max and also when there is no data (e.g. if someone filtered the data dynamically and no datapoints were in the filter range).  Is that true here?  If so, presumably it should be mentioned in the docs.

Reply:
Sure, I can add that once we've decided on a name for the parameter.

Reply:
Both the name and the default value of 2 were copied from bokeh. @jlstevens indicated he didn't like the name in a comment above but we both dislike his only suggestion even more:

>I see. I would like to have a better name for this concept but I don't have one to suggest at the moment. Conceptually it is something like fallback_span but that name is horrible... 

My vote is therefore to stick with ``default_span``.

Reply:
Same here.  Ok, I'll change datashader to 2.0.